pkg/db-repo/orm.go

```diff
@@ -26,25 +25,25 @@ type OrmSettings struct {
 
 func NewOrm(ctx context.Context, config cfg.Config, logger log.Logger, dbClientName string) (*gorm.DB, error) {
 	var err error
-	var connection *sqlx.DB
 
 	if dbClientName == "" {
 		dbClientName = "default"
 	}
 
-	if connection, err = db.ProvideConnection(ctx, config, logger, dbClientName); err != nil {
+	client, err := NewClientGorm(ctx, config, logger, dbClientName)
```

Comment:
```suggestion
	client, err := NewOrmClient(ctx, config, logger, dbClientName)
```

Keep file name and entity names in sync.